packages/babel-preset-gatsby/src/index.js

```diff
@@ -62,19 +64,44 @@ module.exports = function preset(_, options = {}) {
       ],
     ],
     plugins: [
+      [
+        resolve(`@babel/plugin-transform-destructuring`),
```

Comment:
Can you copy over the comment from CRA as well that this is for performance and link to the issue?

Reply:
This change is _probably_ unnecessary since we already use `loose` in the `preset-env`, so I'll remove this.